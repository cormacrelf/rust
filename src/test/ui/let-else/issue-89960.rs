// This had an ICE, see issue #89960

#![feature(let_else)]
#![deny(unused_variables)]

fn main() {
    let Some(ref mut meow) = Some(()) else { return };
    //~^ ERROR unused variable: `meow`
}
