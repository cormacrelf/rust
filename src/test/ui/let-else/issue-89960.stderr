error: unused variable: `meow`
  --> $DIR/issue-89960.rs:7:22
   |
LL |     let Some(ref mut meow) = Some(()) else { return };
   |                      ^^^^ help: if this is intentional, prefix it with an underscore: `_meow`
   |
note: the lint level is defined here
  --> $DIR/issue-89960.rs:4:9
   |
LL | #![deny(unused_variables)]
   |         ^^^^^^^^^^^^^^^^

error: aborting due to previous error

